server/apps/acсount/serializers.py

```diff
@@ -0,0 +1,66 @@
+from rest_framework import serializers
+from .models import User, Team
+
+
+class UserSerializer(serializers.ModelSerializer):
+    full_name = serializers.SerializerMethodField()
+    avatar = serializers.ImageField(required=False, allow_null=True) # 上传图片
+    class Meta:
+        model = User
+        fields = [
+            'id',
+            'username',
+            'full_name',
+            'avatar',
+        ]
+
+    def get_full_name(self, odj) -> str:
+        return f'{odj.first_name} {odj.last_name}'
+
+
+class UserDetailSerializer(serializers.ModelSerializer):
+    full_name = serializers.SerializerMethodField()
+    avatar = serializers.ImageField(required=False, allow_null=True)
+    class Meta:
+        model = User
+        fields = [
+            'id',
+            'username',
+            'full_name',
+            'avatar',
+            'gender',
+            'is_verified',
+            'rating'
+        ]
+
+    def get_full_name(self, odj) -> str:
+        return f'{odj.first_name} {odj.last_name}'
```

Comment:
U have two matching serializers. The only difference between them is more extended list of fields in the second case.

Try creating the `UserBaseSerializer` class and inheriting the `UserSerializer` and `UserDetailSerializer` classes with it.